src/modules/resolver.ts

```diff
@@ -165,7 +173,7 @@ export default class NoteResolver {
         shouldRun = fnPath && !this.getFolderNote(file.parent);
       if (shouldRun && !dryrun) {
         const { path } = fnPath;
-        this.vault.rename(file, path);
+        this.rename(file, path);
```

Comment:
maybe try replacing `this.vault.rename` with `this.plugin.app.fileManager.renameFile` directly?